manifests/manifest_celery_beat_prod.yml

```diff
@@ -1,7 +1,22 @@
 ---
-inherit: manifest_prod.yml
+path: ../
+memory: 1G
+stack: cflinuxfs2
+buildpack: python_buildpack
 env:
+  CACHE_ALL_REQUESTS: True
+  FEC_API_WHITELIST_IPS: true
   NEW_RELIC_APP_NAME: fec | celery beat | prod
+  NEW_RELIC_CONFIG_FILE: /home/vcap/app/newrelic.ini
+  NEW_RELIC_ENV: production
+  NEW_RELIC_LOG: stdout
+  PRODUCTION: True
+  WEB_CONCURRENCY: 4
+services:
+  - fec-api-search
+  - fec-redis
+  - fec-creds-prod
+  - fec-s3-prod
 applications:
   - name: celery-beat
     instances: 1
```

Comment:
Interesting, we have one instance in `prod` but 2 in `dev`?  We may want to take another look at this quick.

@LindsayYoung, should we swap these values or just reduce `dev`?  This is the `celery-beat` app, not the worker.

Reply:
Disregard!  I was mixing up this manifest for the `celery-beat` app with the `celery-worker` app!